galleries/examples/statistics/multiple_histograms_side_by_side.py

```diff
@@ -63,6 +63,12 @@
 
 # %%
 #
+# .. tags::
+#    domain: statistics
```

Comment:
There's a warning in this file

> /home/circleci/project/doc/gallery/statistics/multiple_histograms_side_by_side.rst:101: WARNING: 'any' reference target not found: ../../_tags/domain-statistics-plot-type-barh-plot-type-histogram-styling-position [ref.any]

I suspect this is due to formatting, but not really sure

```suggestion
# .. tags::
#
#    domain: statistics
```